tests/phpunit/api/v3/EventGroupTest.php

```diff
@@ -0,0 +1,73 @@
+<?php
+
+use Civi\Test\HeadlessInterface;
+use Civi\Test\HookInterface;
+use Civi\Test\TransactionalInterface;
+use CRM_MembersOnlyEvent_Test_Fabricator_MembersOnlyEvent as MembersOnlyEventFabricator;
+use CRM_MembersOnlyEvent_Test_Fabricator_Group as GroupFabricator;
+
+/**
+ * EventGroup API Test Case
+ * @group headless
+ */
+class api_v3_EventGroupTest extends \PHPUnit\Framework\TestCase implements HeadlessInterface, HookInterface, TransactionalInterface {
```

Comment:
What is the reason we don't extend HeadlessTest class?

Reply:
This file was generated after running the command: `civix generate:entity EventGroup --api-version=3` 
I've used `civix` version v21.04.1 - the latest version.

I didn't look at it.

Reply:
Should we then use Headlessness, so we don't have to repeat a generic function like setUpHeadless() etc?